doc/rest-api/Client-frontend.md

```diff
@@ -13,68 +13,53 @@ Please see the [Authentication](#authentication) section for more details.
 1. The relevant endpoint has to be configured on the server side.
 See the [configuration section](#configuration).
 1. A list of provided actions is documented with Swagger.
-See the beatiful [specification](http://mongooseim.readthedocs.io/en/latest/swagger/index.html?client=true).
+See the beautiful [specification](http://mongooseim.readthedocs.io/en/latest/swagger/index.html?client=true).
 
 ## Authentication
 
 The only possible authentication method for the time being is *Basic Authentication*.
```

Comment:
Maybe we can replace this with something similar to https://swagger.io/docs/specification/authentication/basic-authentication/ ? Plus remove a separate section for bare jids.


```
MongooseIM uses *Basic Authentication* as an authentication method for the REST API.

Basic authentication is a simple authentication scheme built into the HTTP protocol. The client sends HTTP requests with the Authorization header that contains the word Basic word followed by a space and a base64-encoded string `username@host:password`. Where:

- `username@host` - user's *bare JID*.
- `password` - the password used to register the user's account.

 For example, to authorize as `alice@localhost / secret` the client would send a header:

    Authorization: Basic YWxpY2VAbG9jYWxob3N0OnNlY3JldA==

```

Don't forget to escape the example.

Reply:
I tried to reword it and deleted the bare jids section.